### PR TITLE
Meteor  announcement timing

### LINKED ISSUE #14083: The station (hasn't) cleared the meteor storm.
Meteor storm clearing used to be based on when meteors stopped spawning, but since we made zlevels bigger, it says the station has cleared the wave before the last set of meteors hits.

### LINKED ISSUE #26601: "Station has cleared the meteor storm" is inaccurate
(WEB REPORT BY: west3436 REMOTE: 206.221.180.138:7777)
# Revision
b9a24883bc1f17baaf3c5550993d0f1bc4a4ef75

# Description
This has happened as long as I can remember, but the Galactic Empire Update stating that the station has cleared the meteor storm is comically inaccurate - it usually fires right in the middle of a massive wave of asteroids.

# Steps to Reproduce

# What you Expected

# What Actually Happened

### PR DESCRIPTION
**Pull request by adacovsk**
Closes #26601
Closes #14083

:cl:
 * tweak: Fixed meteor end announcement so the meteors end at the correct time

### CODE REVIEW

File: code/modules/events/meteors.dm [hunk 1]
```diff
@@ -163,7 +162,8 @@ var/global/list/thing_storm_types = list(
 	meteor_wave(rand(10, 20), types = thing_storm_types[storm_name], offset_origin = 150, offset_dest = 230) //Much more clement
 
 /datum/event/thing_storm/end()
-	command_alert("The station has cleared the [storm_name].", "Meteor Alert")
+	spawn(450)
```
Comment (Kurfursten) [inline code/modules/events/meteors.dm]: `45 SECONDS`

also if you're just going to do this to all thing_storms, why not bake in the delay at a higher level and just have the message vary? More maintainable

Comment (adacovsk) [reply code/modules/events/meteors.dm]: Don't know what a thing_storm is so I'd rather not touch it if that's the case

Comment (Kurfursten) [reply code/modules/events/meteors.dm]: You literally just edited it in the latest commit! It's the parent type.

There's also a better implementation for this which is overall less timing naive. You can calculate the end of the storm or get passed it when it finishes.

Comment (adacovsk) [reply code/modules/events/meteors.dm]: Oh I see what you mean now., I'll fix it

Comment (adacovsk) [reply code/modules/events/meteors.dm]: it's not so simple

File: code/modules/events/event_manager.dm [hunk 2]
```diff
@@ -1,7 +1,7 @@
 var/list/allEvents = typesof(/datum/event) - /datum/event
 
-var/eventTimeLower = 6000	//10 minutes
-var/eventTimeUpper = 12000	//20 minutes
+var/eventTimeLower = 6000   //10 minutes
+var/eventTimeUpper = 12000  //20 minutes
```
Comment (Exxion) [inline code/modules/events/event_manager.dm]:
```suggestion
var/eventTimeLower = 10 MINUTES
var/eventTimeUpper = 20 MINUTES
```
I know you didn't write this but might as well clean it up if you're touching it

File: code/modules/events/event_manager.dm [hunk 3]
```diff
@@ -46,7 +46,7 @@ var/scheduledEvent = null
 
 	else if(world.timeofday > scheduledEvent)
 		spawn_dynamic_event()
-
+
```
Comment (Exxion) [inline code/modules/events/event_manager.dm]: ?

**Comment by Kurfursten** [comment]
You really could implement this better and it's not too difficult, but I guess I won't push for it

**Pull request merged**